projects/oku/default.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  pkgs,
+  sources,
+  ...
+}@args:
+
+{
+  metadata = {
+    summary = "Oku is a browser and encrypted data vault based on IPFS";
+    subgrants = [
+      "Oku"
+    ];
+  };
+
+  nixos.modules.programs = {
+    oku = {
+      name = "oku";
+      module = ./programs/oku/module.nix;
+      examples."Enable Oku" = {
+        module = ./programs/oku/examples/basic.nix;
+        tests.basic.module = pkgs.nixosTests.oku;
+      };
+    };
+  };
+
+  nixos.demo.shell = {
+    module = ./programs/oku/examples/basic.nix;
+    description = "oku demo";
```

Comment:
Would be nice to have usage instructions for the demo, since they'll be rendered in the overview.